<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filemanagerdialog</class>
 <widget class="QWidget" name="filemanagerdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2e3440;          /* Темный фон */
    border-radius: 10px;                /* Скругленные углы */
    padding: 20px;
    color: #d8dee9;                    /* Светлый текст */
    font-family: &quot;Segoe UI&quot;, Tahoma, Geneva, Verdana, sans-serif;
    font-size: 14px;
}

QLabel {
    color: #d8dee9;
    font-weight: bold;
    font-size: 16px;
    margin-bottom: 15px;
}

QPushButton {
    background-color: #5e81ac;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    min-width: 80px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #81a1c1;
}

QPushButton:pressed {
    background-color: #4c6a94;
}

QPushButton#buttonCancel {
    background-color: #bf616a;
}

QPushButton#buttonCancel:hover {
    background-color: #d08770;
}

QPushButton#buttonCancel:pressed {
    background-color: #a0494f;
}

</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>351</width>
     <height>247</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnBackUp">
      <property name="text">
       <string>Резервное копирование</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnManager">
      <property name="text">
       <string>Менеджер ФС</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAdmin">
      <property name="text">
       <string>Админ Панель КФС</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMove">
      <property name="text">
       <string>Перемещение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addFiles">
      <property name="text">
       <string>Архивация</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnQuit">
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
